notebooks/example_container_fmriprep_4review.yaml

```diff
@@ -0,0 +1,67 @@
+# This is an example config yaml file for fMRIPrep version 20.2.3.
+
+# Warning!!!
+#   This is only an example, which may not necessarily fit your purpose,
+#   or be an optimized solution for your case,
+#   or be compatible to the fMRIPrep version you're using.
+#   Therefore, please change and tailor it for your case before use it!!!
+
+# Arguments when executing the BIDS App using `singularity run`:
+# You should not have flags of : `--participant-label`, or `--bids-filter-file`!
+babs_singularity_run:
+    -w: "$BABS_TMPDIR"   # this is a placeholder. To be changed to `${PWD}/.git/tmp/wkdir`
+    --n_cpus: '1'
+    --stop-on-first-crash: ""
+    --fs-license-file: "$FREESURFER_LICENSE" # this is a placeholder.
+    --skip-bids-validation: Null  # Null or NULL is also a placeholder
+    --output-spaces: MNI152NLin6Asym:res-2
+    --force-bbr: ""
+    --cifti-output: 91k   # WHEN FULL RUN!
+    -v: '-v'   # this is for double "-v"
+
+
+babs_zip_foldername:   # the foldername(s) to get zipped and the version string(s)
+    fmriprep: "20-2-3"     # should be the same as in `babs-init`'s argument `container_name`; better using "-" instead of "."
+    freesurfer: "20-2-3"   # use the same as in the above line for `fmriprep`
+    # ^^ # ADD THIS WHEN FULL RUN or --anat-only!
+
+cluster_resources:
+    interpreting_shell: /bin/bash   # "-S /bin/bash" on cubic
+    hard_memory_limit: 25G   # "-l h_vmem=25G" on cubic
+    temporary_disk_space: 200G   # "-l tmpfree=200G" on cubic  # this is highly-recommended on cubic
+    customized_text: |
+        #$ -R y
+# You can also add `customized_text` - see docs for more.
+
+# Users need to add their customized bash command below,
+#   they will be used as preambles in `participant_job.sh`
+#   the commands should not be quoted!
+script_preamble: |
+    source ${CONDA_PREFIX}/bin/activate mydatalad    # Penn Med CUBIC cluster; replace 'mydatalad' with your conda env name
+
+# ^^ conda env above: where the scripts generated by BABS will run
+#   not necessary the same one for running `babs-init`
+# ^^ based on what you need on your cluster; some people even don't use `conda`...
+# for MSI, might need to add command e.g., "module_load"
+
+# Where to run the jobs:
+job_compute_space: "${CBICA_TMPDIR}"   # Penn Med CUBIC cluster tmp space
+
+# Below is to filter out subjects (or sessions)
+#   right now we only filter based on unzipped dataset
+required_files:
+    $INPUT_DATASET_#1:
+        - "func/*_bold.nii*"
+        - "anat/*_T1w.nii*"
+
+# Keywords for alerting messages in log files:
+keywords_alert:
+    o_file:
```

Comment:
In the documentation and here, these are referred to "o" and "e", but I think it would be clearer to call them the "stdout" and "stderr" files, since I believe those are the streams that are captured by the two files.

Reply:
I might consult @mattcieslak and @scovitz for this. Usually on SGE the log files are `*.o*` and `*.e*`. I am worried that people won't know `*.o*` files are `stdout` files. 

However I need to confirm with @scovitz that what are the default filenames for these on Slurm clusters? Are they `*.out` and `*.err`? Is it okay to call them `*.o*` and `*.e*` file or better to call them `stdout` and `stderr` file?

Reply:
Better to change `o_file` to `stdout`, and so does `e_file`